Eastern-Research-Group/mywaterway
# Feature/update state national uses

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3399995

## Main Changes:
* Updated the state national uses file.
* Changed the state code for Northern Mariana Islands from "CN" to "MP" to match the states service.

## Steps To Test:
1. Navigate to http://localhost:3000/state/MT/water-quality-overview
2. Make sure the charts load in correctly
3. Navigate to http://localhost:3000/state/MP/water-quality-overview
4. Repeat step 2
